index.html

```diff
@@ -6,17 +6,32 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Portfolio</title>
     <link
-      href="https://fonts.googleapis.com/css2?family=Be+Vietnam+Pro:wght@400;500;700&family=Inter:wght@400;700&family=Poppins:wght@400;700&family=Roboto:ital,wght@0,400;0,700;0,900;1,400&display=swap"
+      href="https://fonts.googleapis.com/css2?family=Be+Vietnam+Pro:wght@400;500;700&family=Inter:wght@400;500;700&family=Poppins:wght@400;700&family=Roboto:ital,wght@0,400;0,700;0,900;1,400&display=swap"
       rel="stylesheet"
     />
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <header class="header-container">
-      <nav class="logo">My Logo</nav>
-      <nav class="nav-bar">
-        <img src="images/Union.png" alt="" >
-      </nav>
-    </header>
+    <div class="container">
+      <header class="header-container">
+        <nav class="logo">My Logo</nav>
+        <nav class="nav-bar">
+          <img src="images/Union.png" alt="" />
```

Comment:
- [ ] please increase the margin righ of the hamburger icon so it fits the original design.